examples/send-atomic-group-complex/main.go

```diff
@@ -0,0 +1,134 @@
+	}
+	log.Println("Connected to Hermez Smart Contracts...")
+	log.Println("Pulling account info from a coordinator...")
+
+	bootCoordNodeState, err := node.GetBootCoordinatorNodeInfo(hezClient)
+	if err != nil {
+		log.Printf("Error obtaining boot coordinator info. URL: %s - Error: %s\n", hezClient.BootCoordinatorURL, err.Error())
+		return
+	}
+	log.Println("Setting actual client ...")
+	hezClient.SetActualCoordinator(bootCoordNodeState.Network.NextForgers[0].Coordinator.URL)
+	log.Println("Actual client is set.")
+
+	log.Printf("Pulling actual coordinator (%s) info...\n", hezClient.ActualCoordinatorURL)
+	actualCoordNodeState, err := node.GetActualCoordinatorNodeInfo(hezClient)
+	if err != nil {
+		log.Printf("Error obtaining boot coordinator info. URL: %s - Error: %s\n", hezClient.BootCoordinatorURL, err.Error())
+		return
+	}
+
+	if debug {
+		log.Printf("Actual coordinator info is: %+v\n", actualCoordNodeState)
+	}
+
+	if len(actualCoordNodeState.Network.NextForgers) > 0 {
+		log.Printf("Actual coordinator URL is: %s\n", actualCoordNodeState.Network.NextForgers[0].Coordinator.URL)
+	}
+
+	if debug {
+		log.Printf("Boot coordinator URL is: %+v\n", actualCoordNodeState.Auction.BootCoordinatorURL)
+	}
+
+	log.Println("Generating BJJ wallet 1...")
+	bjjWallet1, _, err := account.CreateBjjWalletFromHexPvtKey(sourceAccPvtKey1)
+	if err != nil {
+		log.Printf("Error Create a Babyjubjub Wallet from Hexdecimal Private Key. Account: %s - Error: %s\n", bjjWallet1.EthAccount.Address.Hex(), err.Error())
+		return
+	}
+	log.Println("Generating BJJ wallet 2...")
+	bjjWallet2, _, err := account.CreateBjjWalletFromHexPvtKey(sourceAccPvtKey2)
+	if err != nil {
+		log.Printf("Error Create a Babyjubjub Wallet from Hexdecimal Private Key. Account: %s - Error: %s\n", bjjWallet2.EthAccount.Address.Hex(), err.Error())
+		return
+	}
+
+	tx1 := transaction.AtomicTxItem{
+		SenderBjjWallet:       bjjWallet1,
+		RecipientAddress:      "0xf495CC4be6896733e8fe5141a62D261110CEb9F3",
+		TokenSymbolToTransfer: "HEZ",
+		Amount:                big.NewInt(100000000000000000),
+		FeeRangeSelectedID:    126,
+		RqOffSet:              1, //+1
+	}
+
+	tx2 := transaction.AtomicTxItem{
+		SenderBjjWallet:       bjjWallet2,
+		RecipientAddress:      "0x137455AFCD2bEc304E39C988b8BCc66a56FDF32a",
+		TokenSymbolToTransfer: "HEZ",
+		Amount:                big.NewInt(100000000000000000),
+		FeeRangeSelectedID:    126,
+		RqOffSet:              7, //-1
+	}
+
+	txs := make([]transaction.AtomicTxItem, 2)
+	txs[0] = tx1
+	txs[1] = tx2
+
+	// start the txs generation to send
+	atomicGroup := api.AtomicGroup{}
+
+	// create PoolL2Txs
+	atomicGroup.Txs, err = transaction.CreateFullTxs(hezClient, txs)
+	if err != nil {
+		err = fmt.Errorf("[AtomicTransfer] Error generating PoolL2Tx. Error: %s\n", err.Error())
+		log.Printf(err.Error())
+		return
+	}
+
+	// set AtomicGroupID
+	atomicGroup = transaction.SetAtomicGroupId(atomicGroup)
+
+	// Sign the txs
+	for currentAtomicTxId := range txs {
```

Comment:
It's not a `TxID`, but an index of an array

```suggestion
	for i := range txs {
```